src/test/java/io/axoniq/axonserver/connector/command/CommandChannelIntegrationTest.java

```diff
@@ -138,6 +142,34 @@ void testSubscribedHandlersReconnectAfterConnectionFailure() throws Exception {
         assertEquals("", commandResponse.getErrorMessage().getMessage());
     }
 
+    @RepeatedTest(10)
```

Comment:
Did you intend to leave the repeated in? Curious as I personally normally use it locally to make sure it will work more consistently, but then always revert it to a regular `@Test`.

Reply:
I used 100 repetitions locally. I wanted to keep it a repeated test to make it fail more consistently in case of race conditions.
This was the reason for the synchronized disconnect and reconnect methods

Reply:
Gotcha, thanks for pointing that out.